sdk/src/Core/Amazon.Runtime/Pipeline/RetryHandler/RetryPolicy.cs

```diff
@@ -446,27 +445,28 @@ private bool IsClockskew(IExecutionContext executionContext, Exception exception
                     // using accurate server time, calculate correction if local time is off
                     serverTime = serverTime.ToUniversalTime();
                     var diff = correctedNow - serverTime;
+                    var newCorrection = serverTime - realNow;
                     var absDiff = diff.Ticks < 0 ? -diff : diff;
-                    if (absDiff > clockSkewMaxThreshold)
-                    {
-                        var newCorrection = serverTime - realNow;
-                        Logger.InfoFormat(clockSkewMessageFormat,
```

Comment:
nit: What's the rationale for setting this log at INFO level? Just concerned about ROI and log real-estate in some of the consumes such as KTap using SDK calls in multi threaded environment. 

Can it be either a WARN to seek attention from consumer or a DEBUG to indicate the SDK is taking care of such issues.

Reply:
This log helped us debugging the error.